### DISCUSSION TITLE
Rich Text Field | Toolbar icon library ?

### DISCUSSION THREAD

**Discussion by franckmartin** [payloadcms/payload#2122] [Q&A] [answered]
Hi !

From which icon library do the icons used in the rich text editor toolbar come from?

Franck

**Comment by JessRynkar** [comment, marked as answer]
Hi @franckmartin - these are custom icons however you can harvest the SVGs from our codebase [here](https://github.com/payloadcms/payload/tree/master/src/admin/components/icons).

**Reply by franckmartin** [reply]
Thanks @JessChowdhury

**Reply by cyriaqu** [reply]
hello

**Reply by JessRynkar** [reply]
@cyriaqu did you need help with something?
